vassal-app/src/main/java/VASSAL/chat/DynamicClientFactory.java

```diff
@@ -31,11 +31,9 @@ public ChatServerConnection buildServer(Properties param) {
     String url = param.getProperty(URL);
     DynamicClient client = url == null ? new DynamicClient() : new DynamicClient(url);
     if (param.containsKey(DYNAMIC_TYPE)) {
-      //param = new Properties(param);
       // Force the type of the dynamically created server proxy
       param.setProperty(ChatServerFactory.TYPE_KEY, param.getProperty(DYNAMIC_TYPE));
     }
-    client.setOverrides(param);
```

Comment:
Why remove this?

Reply:
The method doesn't do anything and we marked it as deprecated: https://travis-ci.com/github/vassalengine/vassal/builds/177365837#L701